source/main.cpp

```diff
@@ -489,6 +511,8 @@ void PrintHelp()
 	cerr << "    -c, --config <path>: save user's files to given directory." << endl;
 	cerr << "    -d, --debug: turn on debugging features (e.g. Caps Lock slows down instead of speeds up)." << endl;
 	cerr << "    -p, --parse-save: load the most recent saved game and inspect it for content errors." << endl;
+	cerr << "    --parse-images-and-sounds: load all game data, images, and sounds,"
+		" and the latest save game, and inspect data for errors." << endl;
```

Comment:
What is the benefit of also loading the latest save game? And what should happen if there is no latest savegame?